Learning_FEM_1D_step0.ipynb

```diff
@@ -347,38 +347,41 @@
     "If we report our approximate solution in the equation, we get:\n",
     "\n",
     "$$ -\\frac{d}{dx}D\\frac{du_h}{dx} + \\sigma u_h \\ne q \\qquad \\forall x \\in [0,L]$$\n",
+    "(As $u_h$ is an approximation to our solution, we do not expect the equality to hold for every $x$. There should exist some residual function $R(u_h)$ defined as:\n",
```

Comment:
change There should exist some residual function  defined as
TO:
We define the residual as:

Reply:
Changed as per suggestion